src/nls/messages_en.json

```
@@ -66,6 +66,17 @@
     "dashboard.taskRun.status.running": "Running",
     "dashboard.taskRun.status.succeeded": "Completed",
     "dashboard.taskRun.status.waiting": "Waiting",
+    "dashboard.triggerBinding.dateCreated": "Date Created:",
+    "dashboard.triggerBinding.details": "Details",
+    "dashboard.triggerBinding.errorloading": "Error loading TriggerBinding",
+    "dashboard.triggerBinding.labels": "Labels:",
+    "dashboard.triggerBinding.namespace": "Namespace:",
+    "dashboard.triggerBinding.noLabels": "None",
+    "dashboard.triggerBinding.noParams": "No parameters found for this binding.",
+    "dashboard.triggerBinding.title": "Parameters",
```

id is a bit ambiguous / potentially misleading, maybe something more specific would help?

```suggestion
    "dashboard.triggerBinding.parametersTitle": "Parameters",
```